pvfactors/engine.py

```diff
@@ -108,9 +104,11 @@ def fit(self, timestamps, DNI, DHI, solar_zenith, solar_azimuth,
 
         # Fit irradiance model
         self.irradiance.fit(timestamps, DNI, DHI, solar_zenith, solar_azimuth,
-                            surface_tilt, surface_azimuth,
-                            self.params['rho_front_pvrow'],
-                            self.params['rho_back_pvrow'], albedo)
```

Comment:
now the ``rho`` reflectivity values are passed to the irradiance models at initialization: it kind of makes more sense since they are not timeseries values, and bc the ``fit()`` function is more or less for timeseries vectorized calculations